### PR TITLE
Upgrade to next13

### PR DESCRIPTION
**Pull request by aialopes**
Closes https://github.com/i-dot-ai/ellandi/issues/931

### CODE REVIEW

File: web/jest.setup.ts [hunk 1]
```diff
@@ -26,3 +26,8 @@ jest.mock('next/config', () => () => ({
     }
   }
 }))
+
```
Comment (rottitime) [inline web/jest.setup.ts:29]: this is good 👍🏽

File: web/src/pages/signin/forgotten-password/reset.tsx [hunk 2]
```diff
@@ -41,8 +41,13 @@ const ResetPasswordPage = () => {
   }, [isLoading, setLoading])
 
   useEffect(() => {
-    if (!!router.isReady && !code && !user_id) redirectWithError()
-  }, [code, router.isReady, user_id])
+    if (!!router.isReady && !code && !user_id) {
```
Comment (rottitime) [inline web/src/pages/signin/forgotten-password/reset.tsx:44]: @aialopes why can't we use the `redirectWithError()`?

File: web/src/components/Layout/AccountLayout/AccountLayout.tsx [hunk 3]
```diff
@@ -63,14 +63,20 @@ const AccountLayout: FC<Props> = ({
     retry: 0,
     onError: () => {
       invalidate()
-      Router.replace({
```
Comment (rottitime) [inline web/src/components/Layout/AccountLayout/AccountLayout.tsx:66]: @aialopes can you revert back to replace please?

**Review by rottitime — CHANGES_REQUESTED** [review]

**Review by rottitime — APPROVED** [review]

File: web/src/pages/signin/forgotten-password/reset.tsx (hunk 2 above)
Comment (aialopes) [reply web/src/pages/signin/forgotten-password/reset.tsx:44]: redirectWithError wasn't being passed into the useEffect. Decided to refactor.

**Pull request merged**